subprojects/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/report/ClassTestResults.java

```diff
@@ -58,6 +64,17 @@ public String getName() {
         return name;
     }
 
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    public String getReportName() {
+        if (displayName != null && !displayName.equals(name)) {
+            return displayName;
+        }
+        return getSimpleName();
+    }
+
     public String getSimpleName() {
```

Comment:
⭕️ Is this still used anywhere outside? Otherwise it can be private.

Reply:
Done.